src/components/MoviesComponet.js

```
@@ -1,8 +1,13 @@
 import MoviePageHeader from "./MoviePageHeader";
 import MoviepageBody from "./MoviePageBody";
+import MoviepageBody from "./MoviePageBody";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'MoviepageBody'. Consider to delete it or rename it.

'MoviepageBody' is defined here:


(lint/suspicious/noRedeclare)